### PR TITLE
RELEASE.md: add Docker section

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -238,6 +238,30 @@ Create a new release for the newly created Git tag: https://github.com/Icinga/ic
...
+> Only for final versions (not for RCs).
+
+Once the release has been published on GitHub, wait for its
+[GitHub actions](https://github.com/Icinga/icingaweb2/actions) to complete.
+
+```bash
+VERSION=2.9.1
+
+TAGS=(2.9)
+#TAGS=(2.9 2 latest)
```
Comment (nilmerg) [inline RELEASE.md:252]: What's the `2` for? And why is the tag still `2.9` although it's the `2.9.1` we're pulling?

Comment (Al2Klimov) [reply RELEASE.md:252]: In the commented-out case 2.9.1 is the lastest 2.9.x, the lastest 2.x.y and the lastest in general.

Comment (nilmerg) [reply RELEASE.md:252]: So there won't be a tag for a specific minor version? Why that?

Comment (Al2Klimov) [reply RELEASE.md:252]: It will be there once the release GHA are done.

**Pull request merged**